packages/bpk-foundations-web/src/base/buttons.json

```diff
@@ -179,23 +179,23 @@
       "type": "background-image"
     },
     "BUTTON_SECONDARY_DISABLED_BACKGROUND_COLOR": {
-      "value": "{!SKY_GRAY_TINT_06}",
+      "value": "{!BLACK_ALPHA_20}",
```

Comment:
Am wondering if with this file and these tokens are still needed and we should shift to just the private buttons tokens in foundations common 🤔 

I think that is what we want to move to

Reply:
we don't have private tokens for text colour for all buttons though! I used those whenever we had them, otherwise just changed the values of the tokens here

Reply:
Can  we check what we have done on apps? I think for text colours we should just be using the text tokens instead rather than a lot of these custom ones, as these then become available to consumers to use which we want to not have people doing :)

Reply:
Hm yes you're right, let me replace them in that case and will deprecate them